src/main/java/org/kairosdb/core/properties/Props.java

```diff
@@ -0,0 +1,10 @@
+package org.kairosdb.core.properties;
+
+public class Props {
+    public static final String ALIGN_DATAPOINT_TTL_WITH_TIMESTAMP =
```

Comment:
This constant is already defined in here:

https://github.com/InscopeMetrics/kairosdb/blob/master/src/main/java/org/kairosdb/datastore/cassandra/CassandraConfiguration.java#L24